store.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "Makaira Gambio Connect",
+  "repositoryName": "gambio-connect",
+  "shopIdentifier": "gambio-connect",
+  "type": "module",
+  "version": "1.0.7",
+  "title": {
+    "en": "Makaira Gambio Connect",
+    "de": "Makaira Gambio Connect"
+  },
+  "description": {
+    "en": "You can use Makaira with this module.",
+    "de": "Mit diesem Module können sie Makaira nutzen."
+  },
+  "vendor": {
+    "name": "marmalade GmbH",
+    "url": "https://makaira.io/",
+    "avatar": "",
+    "gitlabGroupName": "makaira"
+  },
+  "displayImage": {
+    "en": "/images/placeholder.png",
+    "de": "/images/placeholder.png"
+  },
+  "highlights": [
```

Comment:
Hi @hmennen90 
when installed in one of the Gambio Cloud Shops these benefits do not apply, since Cloud users do not have access to their document root or php files.
Please choose highlights more like "Use the AI-powered Makaira recommendation engine", "Speed up your eShop search" or "Use Makaira with an exclusive low pricing for Gambio shop owners".
Thank you!
Hendrik

Reply:
Hi @FATCHIP-GmbH,

will get these infos today and add it to the store.json

Tanks :-)